### PR TITLE
docs: PDE Update

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
…limitation

## Describe the Change

This PR updates the PDE index page to point to the new tutorial and call out the limitation of PDE and self-hosted Palette.

## Review Changes

💻 [Preview](https://deploy-preview-1179--docs-spectrocloud.netlify.app/devx)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1179#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04.5-devx.md [hunk 1]
```diff
@@ -14,29 +14,25 @@ import InfoBox from 'shared/components/InfoBox';
...
+Palette provides two different modes for deploying and managing applications. The first mode is *Cluster Mode* - this mode enables you to create, deploy, and manage Kubernetes clusters and applications. The second mode is *App Mode* - a mode optimized for a simpler and streamlined developer experience that allows you to only focus on the building, maintenance, testing, deployment, and monitoring of your applications.
 
-You can leverage Spectro Cloud's complimentary managed Kubernetes cluster when using App Mode. The complimentary resources have a limit of 12 vCPU, 16 GiB of memory, and 20 GiB of free storage. Alternatively, you may deploy applications on Kubernetes clusters that belong to your organization and are managed by Palette.
+You can leverage Spectro Cloud's complimentary managed Kubernetes cluster when using App Mode. The complementary resources have a limit of 12 vCPU, 16 GiB of memory, and 20 GiB of free storage. Alternatively, you may deploy applications on Kubernetes clusters that belong to your organization and are managed by Palette.
```
Comment (ritawatson) [inline content/docs/04.5-devx.md:19]:
```suggestion
You can leverage Spectro Cloud's complementary managed Kubernetes cluster when using App Mode. The complementary resources have a limit of 12 vCPU, 16 GiB of memory, and 20 GiB of free storage. Alternatively, you may deploy applications on Kubernetes clusters that belong to your organization and are managed by Palette.
```

File: content/docs/04.5-devx.md [hunk 2]
```diff
@@ -14,29 +14,25 @@ import InfoBox from 'shared/components/InfoBox';
...
-|--------|--------------------------------------------------------------------------------------------------------------------------------------   |
-| 1      |[Login to Palette](https://console.spectrocloud.com/). You may use your Palette credentials or utilize the Social Sign-in capabilities    |
-| 2      |On the right hand-side of the window, click on the User menu. Once the user menu is expanded, click on Switch to App Mode                 |
-| 3      |Select an App Profile (we do offer a set of out-of-the-box profiles) or [Create an App Profile](/devx/app-profile) based on your use case|
-| 4      |Use the App Profile to [deploy your application](/devx/apps)|
+<WarningBox>
 
+App Mode is currently not supported in self-hosted Palette. Contact us if you want to explore App Mode with a self-hosted Palette installation.
+
+</WarningBox>
```
Comment (ritawatson) [inline content/docs/04.5-devx.md:30]: I think may be more suited as an InfoBox since it isn't anything the user can break or or ruin.

**Review by ritawatson — APPROVED** [review]
One typo and a small comment - otherwise, LGTM.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1179#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1179#pullrequestreview-1338054016" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**